dev/doc/xml-protocol-8.5.md

```diff
@@ -0,0 +1,738 @@
+#Coq XML Protocol for Coq 8.5#
+
+This document is based on documentation originally written by CJ Bell
+for his [vscoq](https://github.com/siegebell/vscoq/) project.
+
+Here, the aim is to provide a "hands on" description of the XML
+protocol that coqtop and IDEs use to communicate. The protocol first appeared 
+with Coq 8.5, and is used by CoqIDE. It will also be used in upcoming 
+versions of Proof General.
+
+A somewhat out-of-date description of the async state machine is
+[documented here](https://github.com/ejgallego/jscoq/blob/master/etc/notes/coq-notes.md).
+OCaml types for the protocol can be [found here](https://github.com/coq/coq/blob/v8.5/ide/interface.mli).

```

Comment:
you can replace this line by:

    OCaml types for the protocol can be found in the [`ide/interface.mli` file](/ide/interface.mli).

And the link will work on GitHub while not being dependent on this particular website (cf. example here https://github.com/Zimmi48/coq/blob/patch-2/dev/doc/xml-protocol-8.6.md).